src/components/CardList.js

```diff
@@ -34,30 +33,29 @@ export const CardList = ({ data, navigation, horizontal, refreshControl }) => {
   }
 
   return (
-    <WrapperWithOrientation orientation={orientation}>
-      <FlatList
-        keyExtractor={keyExtractor}
-        data={data}
-        renderItem={({ item }) => (
-          <CardListItem
-            navigation={navigation}
-            horizontal={horizontal}
-            item={item}
-            orientation={orientation}
-            dimensions={dimensions}
-          />
-        )}
-        ListFooterComponent={
-          data.length > 10 &&
-          !listEndReached && (
-            <ActivityIndicator color={colors.accent} style={{ margin: normalize(14) }} />
-          )
-        }
-        onEndReached={() => setListEndReached(true)}
-        refreshControl={refreshControl}
-        style={styles.center}
-      />
-    </WrapperWithOrientation>
+    <FlatList
+      keyExtractor={keyExtractor}
+      data={data}
+      renderItem={({ item }) => (
+        <CardListItem
+          navigation={navigation}
+          horizontal={horizontal}
+          item={item}
+          orientation={orientation}
+          dimensions={dimensions}
+        />
+      )}
+      ListFooterComponent={
+        data.length > 10 &&
+        !listEndReached && (
+          <ActivityIndicator color={colors.accent} style={{ margin: normalize(14) }} />
+        )
+      }
+      onEndReached={() => setListEndReached(true)}
+      refreshControl={refreshControl}
+      style={stylesWithProps({ orientation, dimensions }).wrapper}
```

Comment:
the wrapper component was making problems here, so I added 15% padding style here directly, like centering